service/bin/gunicorn-start

```diff
@@ -9,4 +9,5 @@ export DJANGO_SECRET_KEY="$(tr -dc a-zA-Z0-9 < /dev/urandom | head -c 50)"
 export DJANGO_DEBUG="$(snapctl get django-debug)"
```

Comment:
I'm not that familiar with Django/Gunicorn, but is it okay to set django_secret_key as the test key in the prepare step and have a different one here?

Reply:
It's just outside the diff, but there's a comment:
```
# The secret key is chosen randomly at the start of the gunicorn process
# It is not critical that it remains the same across restarts
# https://docs.djangoproject.com/en/5.1/ref/settings/#std:setting-SECRET_KEY
```